announcements/move-to-cluster-api.md

```diff
@@ -0,0 +1,41 @@
+- Ability to create and manage workload clusters with managed control planes like EKS or AKS.
+- Adding new providers will become easier and Giant Swarm will be in a better position to expand their offering beyond AWS, Azure, and current On-Premises solutions.
+- The possibility to have workload clusters from different providers managed by one management cluster to enable multi-cloud use cases.
+
+### What will the move look like?
+
+Giant Swarm is investing an enormous amount of effort to make the process safe and convenient for you. From your perspective, it will look like a regular update to a new major version -- no workload migration will be required. On the Giant Swarm side, a series of well-tested, automated actions will be performed to transform your current clusters into the Cluster API ones. Giant Swarm wants to provide a transition path for the clusters in all versions and transform them into Cluster API clusters with little to no downtime in the Kubernetes control plane and no downtime in your workloads. Giant Swarm will perform an in-place upgrade to preserve your current IP addresses.
+
+
+In the first phase, starting the transition will be enabled for Giant Swarm personnel only, but eventually, you will be able to perform it on your workload clusters autonomously.
+
+### What are the risks?
+
+The infrastructure behind Giant Swarm clusters differs in many ways from the one backing a Cluster API cluster. Giant Swarm engineers worked hard to find an automated, reliable, and reproducible transition path between the two, but there is always the risk that something goes wrong. Giant Swarm will ensure a well-defined, thoroughly tested transition plan to mitigate the risks and have a recovery plan in place if something unexpected happens. The transition will also be done in phases, leaving your mission-critical cluster to the end to adapt the process to all possible custom configurations and make it bullet-proof.
+
+
+### What will change?
+
+There will be very little to no differences between the Giant Swarm and the Cluster API clusters from the functional point of view.
+
+The most noticeable change will be the deprecation of the 'gsctl' tool that will no longer work for creating or managing Cluster API clusters. You will be able to continue using Giant Swarm Web UI 'Happa' and 'kubectl gs', which support login directly through your company's Single Sign-On with all of its security features provided including multi-factor authentication.
+
+The concept of organizations as a means to isolate different teams, business units, or projects within one installation will remain. Organizations will become more flexible and be mapped to native Kubernetes namespaces and isolation concepts, allowing you to assign users or groups from your identity provider with granular permissions known from [Kubernetes role-based access control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
```

Comment:
```suggestion
The concept of organizations as a means to isolate different teams, business units, or projects within one installation will remain. Organizations will become more flexible and be mapped to native Kubernetes namespaces and isolation concepts, allowing you to assign users or groups from your identity provider with granular permissions known from [Kubernetes role-based access control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).


### What is the ETA?

We have made a big step forward by having 6 of 8 engineering teams within Giant Swarm form a spearhead towards CAPI for the whole month of March. We made good progress, but there's still a lot to do to bring a production-ready and feature-par setup to you. Our next goal is to provide you with an early test version in the coming months and then work in parallel on getting that version ready for production and enable you with a non-breaking upgrade path to it. We will keep you updated on this progress and hope to have something for you soon. In the meantime feel free to reach out to us with questions, concerns, or general comments.
```

adding a note on (non-existent) ETA and our plans. @MartaD WDYT? Once we have this down we should merge I'd say.

Reply:
I'd add 2021 at the end of: for the whole month of March.
I (even post google search) don't know the term feature-par. Suggested change: "...there's still a lot to do to bring you a production-ready setup to you on par with the features you already have."

We will keep you updated on this progress and hope to have something for you soon. Style-wise the 2nd half of the sentence weakens the message. I'd go with: "We will keep you updated as we/things progress."

Reply:
adjusted the above suggestion based on Oshrat's comments 🙏

Reply:
the document was written in 3rd person so I adjusted the suggestion to match the rest. I can rewrite the whole document if you think it's better to use "we/us" not "Giant Swarm"

Reply:
I have no strong feelings about this, except being in full agreement that the voice needs to be consistent across the board.
@HelloHeatherC do you have any thoughts on the preferred voice to use?

Reply:
Oh, I just realized that, it had two different voices, IMHO the preferred voice for a customer announcement would be "we/us" especially if at some point it would be in Slack or sth direct to customers. Using 3rd person "Giant Swarm" is rather Press Release style, which is good, but might be less familiar and friendly to customers. We only use that rarely I think. Happy to hear Heather's thoughts.

Reply:
Strong agree with Puja. Especially when we're talking about what we [deliver/promise](https://copyhackers.com/2012/09/web-copywriting-the-word-we/), it's good to emphasize it's we/us that do the delivering and promising and not some detached formal entity.

Reply:
Ok, I'll rewrite it then